Client/Frontend/Strings.swift

```diff
@@ -1624,6 +1624,47 @@ extension String {
                     comment: "Title for the page where the user sign in to their Firefox Sync account.")
             }
         }
+
+        public struct Search {
+            public static let Title = MZLocalizedString(
+                key: "Settings.Search.PageTitle.v121",
+                tableName: "Settings",
+                value: "Search",
+                comment: "Navigation title for search page in the Settings menu.")
+            public static let ShowSearchSuggestions = MZLocalizedString(
+                key: "Settings.Search.ShowSuggestions.v121",
+                tableName: "Settings",
+                value: "Show Search Suggestions",
+                comment: "Label for show search suggestions setting, in the Search Settings page.")
```

Comment:
Reads better that way?
```suggestion
                comment: "Label for the `show search suggestions` setting, in the Search Settings page.")
```